app/src/main/java/com/kylecorry/trail_sense/astronomy/infrastructure/commands/SunsetAlarmCommand.kt

```diff
@@ -45,88 +45,22 @@ class SunsetAlarmCommand(private val context: Context) : CoroutineCommand {
         val tomorrowSunset =
             astronomyService.getTomorrowSunTimes(location.location, suntimesMode).set
 
-        if (todaySunset != null) {
-            when {
-                isPastSunset(todaySunset) -> {
-                    // Missed the sunset, schedule the alarm for tomorrow
-                    setAlarm(
-                        tomorrowSunset?.minus(alertDuration)
-                            ?: todaySunset.plusDays(1)
-                    )
-                }
+        todaySunset ?: return@onDefault
```

Comment:
This should not return - it changes the logic of rescheduling the alarm to the next day